### PR TITLE
DashBoard Page

### PR DESCRIPTION
**Pull request by reemaRahhal**
Hi Team 2
Our team has built the page with design for web size We have not yet finished responsive design. Please review our code and  we will be happy to receive comments from you for correction or improvement ,

### CODE REVIEW

File: pages/dashboard.html [hunk 1]
```diff
@@ -5,10 +5,267 @@
...
+                <tr></tr>
+                <tr></tr>
+                <tr></tr>
+                <tr></tr> 
+                <tr></tr>
+                <tr></tr>
+                <tr></tr>
+                <tr></tr> 
+                <tr></tr>
+                <tr></tr>
```
Comment (HosenNaser) [inline pages/dashboard.html:217]: unnecessary <tr> tags .. remove the tags and make a style in CSS for this table to get a space between them

File: pages/dashboard.html [hunk 2]
```diff
@@ -5,10 +5,267 @@
...
 </head>
 
 <body>
+    <header>
+        <a class="logo" href="#">my<span>Bank</span></a>
+
+        <img class="img"
+            src="../library/profile.webp"
+            alt="">
+        <i class="bi bi-box-arrow-right , iconLogOut"></i>
```
Comment (HosenNaser) [inline pages/dashboard.html:21]: let this icon disappear if the user is not logged in

**Review by HosenNaser — CHANGES_REQUESTED** [review]
check my recommendation for this page

File: pages/dashboard.html (hunk 1 above)
Comment (reemaRahhal) [reply pages/dashboard.html]: I have already fixed according to your requeste please check

File: pages/dashboard.html (hunk 2 above)
Comment (reemaRahhal) [reply pages/dashboard.html]: Done. I have already fixed the code, please check

Comment (reemaRahhal) [reply pages/dashboard.html]: Done. I have already fixed the code, please check

**Comment by HosenNaser** [comment]
good work @reemaRahhal , just handle the conflicts to merge this branch to master

**Review by HosenNaser — APPROVED** [review]

**Comment by yosef406** [comment]
resolved conflicts

**Pull request merged**